jmxtrans-output/jmxtrans-output-influxdb/src/test/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterTests.java

```diff
@@ -117,7 +119,13 @@ public void pointsAreWrittenToInfluxDb() throws Exception {
 		assertThat(points).hasSize(1);
 
 		Point point = points.get(0);
-		assertThat(point.lineProtocol()).startsWith(lineProtocol);
+		String pointLineProtocol = point.lineProtocol();
```

Comment:
I find this added `pointLineProtocol` variable does not add readability. I much prefer inlining it.